### DISCUSSION TITLE
VS code showing false error how do I stop this?

### DISCUSSION THREAD

**Discussion by ricsirogi** [miguelgrinberg/python-socketio#1154] [Q&A]
For some reason wherever i add @sio.on("event") to my code, VS code assumes that it's an error, even though it's not, and the code runs  just fine. Here's what I mean:
![image](https://user-images.githubusercontent.com/61251735/225407839-91724068-b50d-4205-94ff-34485e950215.png)

I know that this is a problem with my machine, since my friend doesn't have this "error"
I've tried a clean uninstall on VS code, but the issue still persists.
Right Now I only have Jupyter and Pylance installed.

**Comment by miguelgrinberg** [comment]
I'm sorry but I don't know the answer to your question. This looks like a Pylance problem, which I not very familiar with.

**Reply by ricsirogi** [reply]
Where do you think I should ask this question?

**Reply by miguelgrinberg** [reply]
I don't know, sorry. You can ignore the warnings, since obviously they are incorrect.
